samza-kafka/src/main/java/org/apache/samza/system/kafka/KafkaSystemConsumer.java

```diff
@@ -285,6 +321,27 @@ public void register(SystemStreamPartition systemStreamPartition, String offset)
     metrics.registerTopicAndPartition(toTopicAndPartition(tp));
   }
 
+  @Override
+  public void register(SystemStreamPartition systemStreamPartition, Startpoint startpoint) {
```

Comment:
Do you plan to remove the other `register` method once 1.0.1 is released, or do you plan to support the other `register` for more time?